react-ui/src/App.jsx

```diff
@@ -24,6 +24,9 @@ import {
   Settings,
   Submission,
 } from './pages';
+// TODO: Remove when removing pages index file
```

Comment:
It's not clear to me what this TODO means

Reply:
We eventually want to get rid of the index file and import directly, but I guess this is already captured by an issue and doesn't need a TODO.